test/livebook/storage/ets_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule Livebook.Storage.EtsTest do
+  use ExUnit.Case, async: false
+
+  alias Livebook.Storage.Ets
+
+  setup_all do
+    Ets.start_link([])
+
+    :ok
+  end
+
```

Comment:
I don't think we need this anymore?

```suggestion
```

Reply:
In fact we don't but just wanted to keep in here once some other storage backend gets implemented. Will delete it for now if necessary